tests/test_fmi_coupled.py

```diff
@@ -23,11 +23,6 @@
 import pyfmi.fmi as fmi
 from pyfmi.test_util import Dummy_FMUModelME2
 
-try:
```

Comment:
These are unnecessary, the markers are sufficient, i.e., one could test without assimulo using "pytest -m "not assimulo" "